src/Admin/Controllers/OrganizationsController.cs

```diff
@@ -62,7 +68,10 @@ public class OrganizationsController : Controller
         IProviderRepository providerRepository,
         ILogger<OrganizationsController> logger,
         IAccessControlService accessControlService,
-        ICurrentContext currentContext)
+        ICurrentContext currentContext,
+        ISecretRepository secretRepository,
+        IProjectRepository projectRepository,
+        IServiceAccountRepository serviceAccountRepository)
```

Comment:
For the OSS build/running, won't this cause a runtime error when going to the admin portal, since the secret and project repos don't have a Noop implementation?

Reply:
Yes I think it will, great catch. We can just crank out some more noop implementations, although if you have a better idea I would love to hear it. We were talking [above](https://github.com/bitwarden/server/pull/3142#discussion_r1278655937) about combining these repo calls later, is there any way we could reasonably combine these SM-related calls into 1, then we only need 1 noop repo?

Reply:
I would recommend for now just swapping them to fetching <int> and using noop.

Then, when the work on AC-1569 starts, refactor to fetching a DTO via an org repository that has all the information we will need fetched from the DB.